mathesar/urls.py

```diff
@@ -9,6 +9,8 @@
 router.register(r'schemas', viewsets.SchemaViewSet, basename='schema')
 router.register(r'databases', viewsets.DatabaseViewSet, basename='database')
 router.register(r'data_files', viewsets.DataFileViewSet, basename='data-file')
+router.register(r'functions', viewsets.DbFunctionViewSet, basename='functions')
+router.register(r'db_types', viewsets.DbTypeViewSet, basename='db_types')
```

Comment:
I would just call this `types`. We'll have a separate abstractions namespace for Mathesar types.

Reply:
It's ambiguous until the `abstractions` and `db` API namespaces are introduced. I'd change this then.

Reply:
Okay, sounds good.

Reply:
On second thought, why don't we start the `db-api` namespace for new APIs introduced in this PR instead of needing to rename this later?

Reply:
@kgodey maybe the changes related to introducing a new API namespace will be easier to review in a dedicated PR? These endpoints won't be used immediately. I can go either way, I'm just wondering what's faster.